web-portal/components/DatePicker.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <flat-pickr readonly="true" v-model="date" :config="config"></flat-pickr>
+  </div>
+</template>
+
+<script>
+  import flatPickr from 'vue-flatpickr-component'
+  import 'flatpickr/dist/flatpickr.css'
+  import 'flatpickr/dist/themes/dark.css'
+
+  export default {
+    model: {
+      event: 'change'
+    },
+    data: function() {
+      return {
+        date: null,
+        config: {
+          inline: true,
+          enableTime: false,
+          minDate: 'today',
```

Comment:
Small thing we should investigate: with the current setup we allow any selection (including in the past) on the admin page.

Reply:
oh I see, specifically for the admin page? That would be easy enough to implement if we want that

Reply:
fixed 81e13fb2d50ddf4fc16c71420a6756856c629601